doozer/doozerlib/cli/scan_fips.py

```diff
@@ -72,7 +83,53 @@ async def am_i_root(self):
         self.runtime.logger.info(f"Running as user {out}")
         return rc == 0 and "root" in out
 
+    def get_tagged_latest(self, tag):
+        """
+        Returns the latest RPMs and builds, triggered by the automation, tagged in to the brew tag received as input
+        """
+        latest_tagged = self.koji_session.listTagged(tag=tag, latest=True, owner="exd-ocp-buildvm-bot-prod")
+        if latest_tagged:
+            return latest_tagged
+        else:
+            return []
+
+    def get_all_latest_builds(self):
+        brew_tags = []
+        # Setup brew tags
+        tags = self.runtime.get_errata_config()["brew_tag_product_version_mapping"].keys()
+        for tag in tags:
+            major, minor = self.runtime.get_major_minor_fields()
+            brew_tags.append(tag.format(MAJOR=major, MINOR=minor))
+        self.runtime.logger.info(f"Retrieved candidate tags: {brew_tags}")
+
+        tag_rhel_mapping = {}
```

Comment:
This `tag_rhel_mapping` is not used?

Reply:
yeah it was a remnant from the SAST pipeline, forgot to remove. Thank you